src/main/java/com/itkonboarding/airport_gate/dto/request/AirportRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package com.itkonboarding.airport_gate.dto.request;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Airport Dto used for requests
+ *
+ * @author jnejkovic
+ */
+@Data
+public class AirportRequestDto {
+
+    private Integer id;
```

Comment:
Id is not needed in the request objects, since it will be autogenerated. Please remove it from all requests.